app/views/samples/new.html.erb

```diff
@@ -1,3 +1,11 @@
-<%= react_component 'SampleUpload',
-  projects: @projects, csrf: form_authenticity_token, sample: @sample,
-  host_genomes: @host_genomes, loggedin_user: current_user %>
+<div id="sample_upload_page">
+    <%= javascript_tag do %>
+      react_component('SampleUpload', {
+        projects: JSON.parse('<%= raw escape_json(@projects) %>'),
```

Comment:
We still need to find a better solution for this. I know "escape_json" was my suggestion initially :) but it's now clear we need something more native and secure. @trevorb has been busy over the past week, but hopefully we can get some input from him soon